### PR TITLE
Add manual page

### PR DESCRIPTION
**Pull request by wittejm**
This is the feature branch for adding the Manual page.

While @hmarcks and I work on the html / React pieces respectively, we can maintain it has a regular shared branch with individual pull requests rather than pushing to it directly.

### CODE REVIEW

**Comment by wittejm** [comment]
@hmarcks though I did make one more commit just now, to move the header code that was in the Landing page into the shared Header component, so that it can get reused on the Manual page. Hopefully that doesn't interfere with your local changes too much.

**Comment by hmarcks** [comment]
@wittejm sounds good, I'm actually just working from an HTML file I created on my desktop so it's not even local yet :-)

**Comment by hmarcks** [comment]
Actually just finishing up docs now...

**Comment by hmarcks** [comment]
Okay good to go

File: src/frontend/src/components/Manual/index.tsx [hunk 1]
```diff
@@ -0,0 +1,984 @@
...
+                It takes between two and five months for the District Attorney’s office to process your paperwork. If you receive communication during this time from the District Attorney that you would like to review with us, please email us at roe@qiu-qiulaw.com.
+              </li>
+              <li className="mb2">
+                Once your expungement is processed, you will receive paper confirmation from the court informing you that your record was expunged. You should keep copies of this confirmation and make electronic copies – obtaining copies of expunged documents is extremely costly.
+              </li>
+            </ul>
+          </section>
+          <section className="bt bw1 b--light-gray pt2 mb5">
+            <h2 className="f2 fw9 mb3" id="review">
+              Further Analysis Needed
```
[the remark this answers is not in the document]
Comment (hmarcks) [reply src/frontend/src/components/Manual/index.tsx:498]: Just to confirm, if Michael agrees, all of them can be removed?

Then we don't have any scenarios where the charge needs a review and doesn't have an input to clarify it?

[the remark this answers is not in the document]
Comment (wittejm) [reply src/frontend/src/components/Manual/index.tsx:498]: I checked with Michael and he agreed we can delete this section.

**Comment by wittejm** [comment]
I'm about to push these minor changes

**Comment by hmarcks** [comment]
Do you mind if I do? I have some other updates so would like to avoid conflicts

**Comment by wittejm** [comment]
sure, sounds good

**Comment by wittejm** [comment]
Ok. I rebased (and meanwhile borked then un-borked the commit history), and added one last React logic adjustment to the Header (with no change in behavior). I can ask @KentShikama to look over the react code but otherwise if you're satisfied I can merge

**Comment by hmarcks** [comment]
Good on my end, thanks!

File: src/frontend/src/components/App/index.tsx [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
-import Header from '../Header';
 import Footer from '../Footer';
 import Authenticator from '../Authenticator';
 
 class App extends React.Component {
   public render() {
     return (
       <Authenticator>
-        <Header />
+        {/*<Header /> // added to individual pages and excluded from others */}
```
Comment (KentShikama) [inline src/frontend/src/components/App/index.tsx:10]: What is this comment?

**Pull request merged**